release/tag_repos.sh

```diff
@@ -8,6 +8,7 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
```

Comment:
Just to check - I think enabling the debugging output was deliberate, yes? (as you also added the /dev/null redirects etc).

Reply:
yes, removing now.